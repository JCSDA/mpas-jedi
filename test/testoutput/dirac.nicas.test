Test     : Increment norm: 7568.15
Test     : Increment norm: 5291.59
Test     : Adjoint test: 3264.82 / 3264.82

