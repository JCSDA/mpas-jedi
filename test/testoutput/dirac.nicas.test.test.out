Test     : Increment norm: 0.0111366
Test     : Increment norm: 0.00463284
